# Generated for NaoQi method ALFrameManager.setTimelineFps
# Sets the FPS of a given timeline

# The id of the timeline.
string id
# The FPS to set.
int32 fps
---


